### DISCUSSION TITLE
Feedback for layout design

### DISCUSSION THREAD

**Discussion by LaraBiss** [Oeffis/Oeffis#54] [Ideas]
Related issue: #33

[Figma Link](https://www.figma.com/file/y1MLDz9KPjwKa6MqFeEdIp/Prototyp?type=design&node-id=0%3A1&t=JKm7Xb5xhiMlm6zH-1) (no registration needed)

Here is our first wireframe for the general desktop layout. 
Please have a look at the rough sketch of route planning and post your feedback in this discussion.
The yellow post-it's provide additional information and uncertainties. You can ignore the green post-it's.

**Comment by timvahlbrock** [comment]
@Oeffis/core-team

**Comment by timvahlbrock** [comment]
If I get this correctly, the Layout is currently heavily google maps inspired - which is somewhat good as the objective was to build "google maps for public transport". But as google maps does have public transit support, we should have an eye on what currently disturbs us when using it and - in consequence - how our layout should be different to try to improve this.

**Reply by timvahlbrock** [reply]
When thinking about this, I think the things we want to do different than maps-based applications are mostly about the underlying data, and features added later, right? So maybe there isn't much to modify currently.

**Reply by JaJako** [reply]
Regarding things we could do different than other map-based solutions, me personally would find it handy to have some mechanism (for example some sort of slider separating map and route planning details) that let's you adjust how much space each part takes on your screen. A big map is good in general, but if you want to study the details of your planned route, it would be convenient being able to have that part bigger, too. Especially on large, high-resolution displays such sidebars become relatively small. (Of course that is something relevant to the Desktop version only.)

**Reply by LaraBiss** [reply]
The slider sounds like a good idea. Definetly noted for further improvement and we should keep this in mind while implementing the first functional version. 
I think the main differences in the UI will occur when we have more features to implement like the history we started to add. If we focus on route planning for now the UI will be close to google maps because the only two components are planning details and the map. But we'll reconsider the look of planning details (and we also wanted to include a version of trains with delayed arrival).

**Comment by LaraBiss** [comment]
The mobile version [click](https://www.figma.com/file/O7dJ1NyevhYLitnOwRnI6M/Mobile-Prototyp?type=design&node-id=0%3A1&t=t1Cotrdj1NSd9jxO-1). 

@Oeffis/core-team

**Reply by timvahlbrock** [reply]
This link is not publicly accessible

**Reply by LaraBiss** [reply]
I updated the link, now you should have permission: [click](https://www.figma.com/file/O7dJ1NyevhYLitnOwRnI6M/Mobile-Prototyp?type=design&node-id=0-1&t=ePnGMyA9zv0EpOCo-0)
I also added it in the issue.
